templates/bundles/DemosPlanCoreBundle/UI/button.html.twig

```diff
@@ -80,6 +80,10 @@
         disabled
     {% endif %}
 
+    {% if dataCy is defined %}
```

Comment:
If we're using `attributes`, we do not need the extra definition here.

Reply:
Done in https://github.com/demos-europe/demosplan-core/pull/1742/commits/60c5c909b95dfc5f604b18f71bbec1c4c4a6755b
Thanks!